paper-tabs.html

```diff
@@ -285,24 +278,38 @@
         notify: true
       },
 
+      selectable: {
+        type: String,
+        value: 'paper-tab'
+      },
+
       _step: {
         value: 10
       },
 
       _holdDelay: {
         value: 1
-      },
-
-      _tab: {
-        observer: '_tabChanged'
       }
+    },
 
+    hostAttributes: {
+      role: 'tablist'
+    },
+
+    ready: function() {
+      // TODO(cdata): Remove this when Polymer fixes behavior application
+      // order issues:
+      this.setAttribute('role', 'tablist');
```

Comment:
No, because it was fixed by the time I noticed it was a problem, just not tagged yet.